mps-model-adapters-plugin/src/test/kotlin/org/modelix/model/mpsadapters/MpsAdaptersTestBase.kt

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2023.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.modelix.model.mpsadapters
+
+import com.intellij.ide.impl.OpenProjectTask
+import com.intellij.openapi.Disposable
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.project.ProjectManager
+import com.intellij.openapi.project.ex.ProjectManagerEx
+import com.intellij.openapi.util.Disposer
+import com.intellij.testFramework.TestApplicationManager
+import com.intellij.testFramework.UsefulTestCase
+import jetbrains.mps.ide.ThreadUtils
+import jetbrains.mps.ide.project.ProjectHelper
+import jetbrains.mps.project.MPSProject
+import java.nio.file.Files
+import java.nio.file.Path
+import kotlin.io.path.absolute
+import kotlin.system.measureTimeMillis
+
+/**
+ * Based on org.jetbrains.uast.test.env.AbstractLargeProjectTest
+ */
+@Suppress("removal")
+abstract class MpsAdaptersTestBase : UsefulTestCase() {
+
+    protected lateinit var project: Project
+
+    override fun runInDispatchThread() = false
+
+    override fun setUp() {
+        super.setUp()
+        TestApplicationManager.getInstance()
+        val projectOpenTime = measureTimeMillis {
+            project = openTestProject()
+        }
+        LOG.warn("Project has been opened successfully in $projectOpenTime ms")
```

Comment:
```suggestion
        LOG.warn("Project has been opened successfully [timeMs=$projectOpenTime]")
```

Or with placeholders if supported by the logger here.

Cf. https://issues.modelix.org/articles/MODELIX-A-67/2024-02-22-Coding-conventions-workshop#logging

Reply:
Removed the time measurement and logging. Was copied from some other implementation, but we don't need it.